app/src/main/res/values/strings.xml

```diff
@@ -526,16 +532,18 @@
   <string name="bloodsugarsummary_bloodsugartype_ppbs">PPBS</string>
   <string name="bloodsugarsummary_bloodsugartype_fbs">FBS</string>
 
+  <!-- Blood Pressure History screen -->
   <string name="bloodpressurehistory_bp_reading" translatable="false">%1$d / %2$d</string>
   <string name="bloodpressurehistory_add_bp_button">Blood Pressure</string>
   <string name="bloodpressurehistory_toolbar_title" translatable="false">%1$s, %2$s, %3$d</string>
   <string name="bloodpressurehistory_bp_time_date" translatable="false">%1$s\n%2$s</string>
 
-  <string name="bloodpressuresummaryview_see_all_button">See all</string>
-  <string name="patientsummary_newbp_date_time" translatable="false">%1$s\n%2$s</string>
-
+  <!-- Blood Pressure History screen -->
```

Comment:
Isn't supposed to be Blood Sugar History Screen?

Reply:
Whoops, will fix!